### ISSUE TITLE
prometheus-metrics-ghc-0.3.0 build failure with ghc 8.4: GCStats not in scope

### ISSUE THREAD

**Issue by DanBurton** [fimad/prometheus-haskell#39]
As seen on the Stackage build server:

```
[1 of 1] Compiling Prometheus.Metric.GHC ( src/Prometheus/Metric/GHC.hs, dist/build/Prometheus/Metric/GHC.o )

src/Prometheus/Metric/GHC.hs:131:54: error:
    Not in scope: type constructor or class ‘GCStats’
    |
131 |                => String -> String -> SampleType -> (GCStats -> a) -> IO [SampleGroup]
    |                                                      ^^^^^^^
```

I was able to reproduce this locally like so:

```bash
stack unpack prometheus-metrics-ghc-0.3.0 && cd prometheus-metrics-ghc-0.3.0
edit stack.yaml # add the following stack.yaml
stack build --test --bench --no-run-benchmarks
```

```yaml
# stack.yaml
resolver: nightly-2018-07-04
extra-deps:
- prometheus-client-0.3.0
```

**Comment by ocharles** [comment]
Thanks, I think the open PRs should fix this, so we should be able to get this fixed soon.

**Comment by purcell** [comment]
Yep, this is addressed by #38.

**Comment by ocharles** [comment]
#38 is merged, I'll be cutting a new release shortly.
